velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -665,6 +732,39 @@ class ExpressionFuzzer {
 
  public:
   void go() {
+    // Go to on-disk repro mode.
```

Comment:
Do we need to couple the repro logic with the Fuzzer? Would it be more flexible to create a separate CLI that takes a data and SQL paths and executes the expression o the data?

Reply:
I think it is better to keep them together because we need to see the discrepancy of common path vs simple path when reproducing. For example when a fuzzer run fail due to result not matching, then we need to compare the results  between common and simple path to debug.

Reply:
Maybe we can have this functionality available separately as well. For example, we could add a flag to tell whether to use common or simplified expression evaluation path.

One concern about putting everything into the Fuzzer is it makes the Fuzzer very large ands dependencies such as DuckDB.

Reply:
Thanks, that makes sense. Quick question, what is the downside of making Fuzzer a little larger?